src/utils/utilities.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
I like the abstraction, happy with it as it sits but if I was to give feedback...  I would expect a function named `percent` to just return the percent - or NaN if an input was incorrect. So this could be 2 functions, `percentOfPlayerWins` which calls `percent` to get the percent value and handles the additional logic.

also... I know it's my code originally, but as this sits we can make this more succinct..

```js
let total = wins + losses;
return total > 9 ? (Math.round(wins / (total) * 100) || 0) : '-';
```

:smiley: